.gitignore

```diff
@@ -2,6 +2,7 @@ target/
 /jsign-gradle-plugin/.gradle
 /jsign-gradle-plugin/build
 dependency-reduced-pom.xml
+/jsign-core/src/test/resources/keystores/privatekey.pkcs1.pem.key

```

Comment:
One last thing, the file should be copied in the `target/test-classes/keystores` directory, it will be removed automatically by `mvn clean` and there is no need to declare it in `.gitignore`

Reply:
Right, this is a leftover from when the shell script copied the file. I've amended my squashed commit to remove this :+1: